core/controller/block.ts

```diff
@@ -14,18 +14,39 @@ import { ActionTypes } from 'core/util/actionTypes';
 import { IKeyPair } from 'shared/util/ed';
 import SharedTransactionRepo from 'shared/repository/transaction';
 import TransactionQueue from 'core/service/transactionQueue';
+import { VersionChecker } from 'core/util/versionChecker';
+import { PeerAddress } from 'shared/model/types';
+import PeerMemoryRepository from 'core/repository/peer/peerMemory';
 
 interface BlockGenerateRequest {
     keyPair: IKeyPair;
     timestamp: number;
 }
 
+type BlockReceiveData = {
+    data: BlockModel,
```

Comment:
This `data` can be renamed as a `blockSchema` or `blockModel`. I think we can to name variable as `data` if we can't definitely name it.